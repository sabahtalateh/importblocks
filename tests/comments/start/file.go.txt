package test1

import(
	// GITHUB.COM/A/B
	"github.com/a/b"
	"os"
	"x/y/z"
	// IMPORTBLOCKS.B
	"github.com/sabahtalateh/importblocks/b"
	"fmt"

	"github.com/sabahtalateh/importblocks/c"
	"github.com/sabahtalateh/importblocks/a"
	"bytes"
)

// MATH
import "math"
