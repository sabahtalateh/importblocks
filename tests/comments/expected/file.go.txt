package test1

import (
	"bytes"
	"fmt"
	"math"
	"os"

	"github.com/sabahtalateh/importblocks/a"
	// IMPORTBLOCKS.B
	"github.com/sabahtalateh/importblocks/b"
	"github.com/sabahtalateh/importblocks/c"

	// GITHUB.COM/A/B
	"github.com/a/b"

	"x/y/z"
)
